twingate/internal/provider/resource/remote-network.go

```diff
@@ -30,6 +31,13 @@ func RemoteNetwork() *schema.Resource {
 				Required:    true,
 				Description: "The name of the Remote Network",
 			},
+			"location": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice(model.Locations, false),
+				Description:  "The location of the Remote Network",
```

Comment:
@alexmensch please check description

Reply:
We should list out the possible options here. So let's change it to: The location of the Remote Network. Must be one of the following: AWS, AZURE, GOOGLE_CLOUD, ON_PREMISE, OTHER